### PR TITLE
feature: add JSON feed for plugins

### PR DESCRIPTION
**Pull request by ryangjchandler**
Feed can be reached at `/plugins/feed`. Pretty simple structure, includes a `last_updated_at` value in the response so that caching can be performed client side / server side when requesting data from the feed.

There's no filtering or anything, it's all very barebones. My plan is to use this with a Sushi model.

### CODE REVIEW

File: app/Http/Controllers/Plugins/FeedController.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        $plugins = Plugin::query()
+            ->published()
+            ->with(['author', 'media'])
+            ->get()
+            ->map(fn (Plugin $plugin): array => [
+                'id' => $plugin->id,
+                'name' => $plugin->name,
+                'slug' => $plugin->slug,
+                'description' => $plugin->description,
+                'thumbnail' => $plugin->getThumbnailUrl(),
```
Comment (pxlrbt) [inline app/Http/Controllers/Plugins/FeedController.php:22]: Should we call this `thumbnail_url`?

**Review by pxlrbt — COMMENTED** [review]
Feel like this should be on a separate `/api` endpoint and be the `PluginController` then

File: app/Http/Controllers/Plugins/FeedController.php (hunk 1 above)
Comment (ryangjchandler) [reply app/Http/Controllers/Plugins/FeedController.php:22]: Probably, yeah.

**Comment by SebastiaanKloos** [comment]
Would be cool if you can also filter on plugin vendor

**Comment by ryangjchandler** [comment]
@SebastiaanKloos I think that should all be done in userland.

**Comment by danharrin** [comment]
Rerouted to /feed/json in case we introduce RSS in the future

**Pull request merged**